dex/src/test/scala/com/wavesplatform/dex/market/OrderBookActorSpecification.scala

```diff
@@ -614,4 +614,10 @@ class OrderBookActorSpecification
       afsIsNotEnoughTest(marketOrderType = OrderType.BUY, feeAsset = Waves, isFeeConsideredInExecutionAmount = true)     // fee in spent asset
     }
   }
+
+  private def getAggregatedSnapshot(orderBookRef: ActorRef): OrderBookAggregatedSnapshot = {
+    val pair       = wavesUsdPair // hack
```

Comment:
?

Reply:
It is a hack to get `OrderBookAggregatedSnapshot` through `OrderBookAskAdapter` that requires pair. I could move pair to the arguments, but there are no reasons for this right now.